esphome/__main__.py

```diff
@@ -545,6 +555,10 @@ def parse_args(argv):
         help="Manually specify a serial port to use"
         "For example /dev/cu.SLAB_USBtoUART.",
     )
+    parser_logs.add_argument(
+        "--ca-cert",
```

Comment:
```suggestion
        "--mqtt-ca-cert",
```

If this is just for the mqtt module, it should be prefixed with mqtt.